core/src/components/select/select.tsx

```diff
@@ -665,13 +668,13 @@ export class Select implements ComponentInterface {
    * was passed.
    */
   private get labelText() {
-    const { el, label } = this;
+    const { label } = this;
 
     if (label !== undefined) {
       return label;
     }
 
-    const labelSlot = el.querySelector('[slot="label"]');
+    const labelSlot = this.labelSlot;
```

Comment:
~ Should we match the code style above and do something similar to:

```ts
const { labelSlot } = this;
```

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/27483/commits/13a6547db14ec4d78c1b59c8f000014f5e0ebecb